### ISSUE TITLE
ansible-console does not become

### ISSUE THREAD

**Issue by hyperized** [ansible/ansible#16614]
##### ISSUE TYPE
- Bug Report
##### ANSIBLE VERSION

```
ansible 2.1.0.0
```
##### CONFIGURATION

We're using ec2.py to gather inventory
##### OS / ENVIRONMENT

n/a
##### SUMMARY

ansible-console seems to be unable to become root, even though the sudo password is prompted and specific instructions are given to become and to become user.
##### STEPS TO REPRODUCE

```
$ ansible-console
SUDO password:
Welcome to the ansible console.
Type help or ? to list commands.

myuser@all (93)[f:50] cd tag_groug_serrver
myuser@tag_group_server (1)[f:50]$ become yes
myuser@tag_group_server (1)[f:50]$ whoami
[redacted] | SUCCESS | rc=0 >>
myuser

myuser@tag_group_server (1)[f:50]$ become_user root
myuser@tag_group_server (1)[f:50]$ become yes
myuser@tag_group_server (1)[f:50]$ whoami
[redacted] | SUCCESS | rc=0 >>
myuser

myuser@tag_group_server (1)[f:50]$ become yes
myuser@tag_group_server (1)[f:50]$ become_user root
myuser@tag_group_server (1)[f:50]$ whoami
[redacted] | SUCCESS | rc=0 >>
myuser
```
##### EXPECTED RESULTS

I expected behavior like:

```
$ ansible tag_group_server -m shell -a "whoami" --become --ask-become-pass
SUDO password:
[redacted] | SUCCESS | rc=0 >>
root
```
##### ACTUAL RESULTS

Not being root when using become and become_user on the ansible-console.
The order in which the commands are given seems not to matter.

**Comment by bcoca** [comment]
Fixed by 11eefdc4866a120f2e4805115947de6345885194

f you continue seeing any problems related to this issue, or if you have any further questions, please let us know by stopping by one of the two mailing lists, as appropriate:

https://groups.google.com/forum/#!forum/ansible-project - for user questions, tips, and tricks
https://groups.google.com/forum/#!forum/ansible-devel - for strategy, future planning, and questions about writing code
Because this project is very active, we're unlikely to see comments made on closed tickets, but the mailing list is a great way to ask questions, or post if you don't think this particular issue is resolved.

Thank you!

**Comment by hyperized** [comment]
Thank you, can you tell me in which release this will likely be fixed?

**Comment by bcoca** [comment]
it will be released in 2.1.1
